core/matching/matchers/xml_templated_match.go

```diff
@@ -0,0 +1,102 @@
+package matchers
+
+import (
+	"regexp"
+
+	"github.com/beevik/etree"
+)
+
+var XmlTemplated = "xmltemplated"
+
+var ignoreExpr = regexp.MustCompile("^\\s*\\{\\{\\s*ignore\\s*\\}\\}\\s*$")
+var regExpr = regexp.MustCompile("^\\s*\\{\\{\\s*regex:(.*)\\}\\}\\s*$")
```

Comment:
The Hoverfly response template function is in this format: `{{ function_name arg1, arg2.. }}`
So I would suggest replacing the `:` with space as delimiter such as `{{ regex your_regex_string_value }}` for consistency.

Reply:
Thank you for merged it, and sorry for long time no commits.
I am a little busy recently. After we finish our tasks, I will add a commit for this.

Reply:
👍  whenever you have some time.